### ISSUE TITLE
Scrolling bug when calendar view extends outside of safe area

### ISSUE THREAD

**Issue by hiltonc** [airbnb/HorizonCalendar#90]
I have the calendar view extending to top and bottom edges of the screen:

```
NSLayoutConstraint.activate([
    calendarView.leadingAnchor.constraint(equalTo: view.layoutMarginsGuide.leadingAnchor),
    calendarView.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor),
    calendarView.topAnchor.constraint(equalTo: view.topAnchor),
    calendarView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
])
```

When I scroll to the end it glitches:

```
calendarView.scroll(
            toDayContaining: Date(),
            scrollPosition: .centered,
            animated: true)
```

https://user-images.githubusercontent.com/470753/105067755-e57e7000-5a34-11eb-9b38-f5cb892b3982.mov

If I constrain the top and bottom to the `layoutMarginsGuide` I don't have this issue. I'd prefer to extend to the edges of the screen so that I get the content under the navigation bar and home indicator visual effect.

**Comment by bryankeller** [comment]
Oh yikes! Definitely a bug, thanks for reporting. I'll look into this soon.

**Comment by bryankeller** [comment]
@hiltonc fix will be merged Monday at the latest, thanks for reporting this bug!

https://github.com/airbnb/HorizonCalendar/pull/101

**Comment by hiltonc** [comment]
Awesome, thank you @bryankeller!

**Comment by bryankeller** [comment]
Fixed in v1.7.1 🙂
